dm/master/election.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2019 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package master
+
+import (
+	"context"
+
+	"go.uber.org/zap"
+
+	"github.com/pingcap/dm/pkg/log"
+)
+
+func (s *Server) electionNotify(ctx context.Context) {
+	for {
+		select {
+		case <-ctx.Done():
+			return
+		case leader := <-s.election.LeaderNotify():
+			// output the leader info.
+			if leader {
+				log.L().Info("current member become the leader", zap.String("current member", s.cfg.Name))
+			} else {
+				_, leaderID, err2 := s.election.LeaderInfo(ctx)
+				if err2 == nil {
+					log.L().Info("current member retire from the leader", zap.String("leader", leaderID), zap.String("current member", s.cfg.Name))
+				} else {
+					log.L().Error("get leader info", zap.Error(err2))
```

Comment:
Is `Warn` level better

Reply:
updated in https://github.com/pingcap/dm/pull/367/commits/708c2c1288d3bd3a379399d6e22f76572488b9fb.